src/gtest/test_checktransaction.cpp

```diff
@@ -816,7 +814,7 @@ TEST(ChecktransactionTests, SaplingSproutInputSumsTooLarge) {
     {
         // create JSDescription
         uint256 rt;
-        uint256 joinSplitPubKey;
+        Ed25519VerificationKeyBytes joinSplitPubKey;
```

Comment:
An Ed25519 verification key is a byte sequence by design, so the `Bytes` is redundant.

Reply:
This was to match the naming in `ed25519-zebra`, where they have separate byte and stateful types, but I'm fine with renaming it on the C++ side.